<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
<title>DISTANCE-HI-C</title>
<meta http-equiv="content-type" content="text/html; charset=utf-8" />
<link rel="stylesheet" type="text/css" media="screen" href="screen.css" />
<!--[if lt ie 7]><link rel="stylesheet" type="text/css" media="screen" href="ie-win.css" /><![endif]-->
</head>
<body id="bmission">
<div id="container">
  <div id="header">
    <h1 style="font-size:18px"><a href="#">DISTANCE-HI-C</a></h1>
    <h2 style="font-weight:bold; ">CREATING PHOTO ACTIVATED X-LINKERS TO DEFINE NUCLEAR ARCHITECTURE</h2>
  </div>
  <div id="page-header"> <img src="cornell-bird.jpg" width="780px" alt="" />
    <ul>
      <li><a href="#">Abstract</a></li>
      <li><a href="#publications">Publications</a></li>
      <li><a href="#aboutus">About Us</a></li>
    </ul>
  </div>
  <div id="wrapper" style="text-align:left; font-size:16px">
      <h3>Abstract</h3>
<p style="text-align:left; font-size:16px">
Nuclear organization of DNA is complex and consists of multiple layers. At the lowest resolution, large sections of chromosomes are packed in territories, at a higher level chromosomal regions are organized in topologically-associated domains that provide a framework for interactions of transcription factors (TFs) bound to promoters and distal regulatory elements (enhancers), and finally the structural interplay of regulators, RNA polymerase II, and chromatin then lead to regulated gene expression. In recent years, a number of methods called chromatin (conformation) capture (CC) have been developed and used to capture dynamic and stable DNA contacts that constitute genome architecture and potential regulatory interactions. A major limitation of these methods is that they all depend on a single crosslinker, formaldehyde, which crosslinks DNA to proteins as well as proteins to other proteins. This complicates the interpretation of the observed 'DNA-DNA' contacts and it lacks distance information.
</p>
<p style="text-align:left; font-size:16px">
Here we propose an orthogonal strategy, named Distance-Hi-C (D-Hi-C), where we design, test, and apply a battery of photo-activated crosslinkers, designed to directly measure distances between interacting sites genome-wide. These bivalent crosslinkers consist of two reactive groups separated by a linker. DNA intercalaters that can be crosslinked by photo-activation (i.e. Psoralen) will be used as the reactive groups. Linkers will be of varying precise lengths, and either flexible or rigid in nature so that the 3-dimensional distance between crosslinked loci can be inferred. Such DNA-specific bivalent crosslinking reagents, when substituted for formaldehyde in Hi-C protocol, produces space constraints revealing enhancer- promoter interactions and potentially allowing the inference of the 3D arrangement of the nuclear genome with unprecedented precision. Moreover, D-Hi-C is expected to lower backgrounds and allow examination of short and moderate range interactions, which are obscured by high backgrounds of current methods. Addition of groups such as digoxigenin to the bivalent crosslinkers, in addition to biotin incorporated to the ligation products, will enable better purification and thus deeper examination of genomic interactions. Finally, sampling DNA distances in time following gene activation provides a means of exploring the 4D architecture and setting critical limits in evaluating mechanisms of gene activation. The specific aims are to
</p>
<ul style="text-align:left; font-size:16px; color:black">
<li>1) Synthesize a battery of bivalent crosslinkers and evaluate their ability to crosslink DNA in vitro; </li>
<li>2) Test D-Hi-C crosslinkers relative to formaldehyde in the Drosophila nuclei model; </li>
<li>3) Apply new crosslinkers to tier 1, ENCODE GM12878 and K562 cell lines to test their efficacy in human cells; </li>
<li>4) Test locus-specific photo-crosslinking that will allow a more focused and thorough examination of locus-specific interactions with the genome in a time course of gene activation. </li>
</ul>
<p>
The development of these methodologies will be broadly useful in providing critical insights into gene regulatory mechanisms that are operative in normal animal development and homeostasis and that go awry in diseases like cancer.
</p>

<h3 id="publications">Publications</h3>
<p style="text-align:left; font-size:16px">
1. Wang Z, Martins AL, Danko CG (2016).  RTFBSDB: an integrated framework for transcription factor binding site analysis.
Bioinformatics.  In press.( Preprint: <A href="http://biorxiv.org/content/early/2016/01/05/036053.abstract" style="font-weight:normal">http://biorxiv.org/content/early/2016/01/05/036053.abstract</A>)
</p>
<h3 id="aboutus">About Us</h3>
<p style="text-align:left; font-size:16px">
<div ><IMG src="cornell.logo.png" style="vertical-align:middle" width="24px" height="24px"/> &nbsp;&nbsp;<A target="_blank" href="https://blogs.cornell.edu/johnlislab/" style="font-size:20px;text-align:">John Lis Lab</A></DIV>
<div ><IMG src="cornell.logo.png" style="vertical-align:middle" width="24px" height="24px"/> &nbsp;&nbsp;<A target="_blank" href="http://www.dankolab.org/"style="font-size:20px">Charles Danko Lab</A></DIV>
<div ><IMG src="cornell.logo.png" style="vertical-align:middle" width="24px" height="24px"/> &nbsp;&nbsp;<A target="_blank" href="http://lin.chem.cornell.edu/" style="font-size:20px">Hening Lin Lab</A></DIV>
<div ><IMG src="cornell.logo.png" style="vertical-align:middle" width="24px" height="24px"/> &nbsp;&nbsp;<A target="_blank" href="https://www.bme.cornell.edu/people/profile.cfm?netid=wrz2"style="font-size:20px">Warren Zipfel Lab</A></DIV>
</p>
    </div>
  <div id="footer">
	<hr/>
    <p> Copyright &copy; 2016 Cornell University. All rights reserved. </p>
    <p> Design by <a href="http://www.dankolab.org/">Danko Lab.</a> [Web template from <a href="http://www.free-css.com">Free CSS</A>.]</p>
  </div>
</div>
</body>
</html>
